benchmarks/bench_gap_encoder_hp.py

```diff
@@ -13,10 +13,11 @@
 from skrub import GapEncoder
 import seaborn as sns
 import matplotlib.pyplot as plt
+import loguru
```

Comment:
The standard usage is rather
```suggestion
from loguru import logger
```